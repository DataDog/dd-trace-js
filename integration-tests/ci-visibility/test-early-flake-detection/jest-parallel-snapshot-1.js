'use strict'

describe('parallel snapshot', () => {
  it('can do multiple snapshots', () => {
    expect('hello').toMatchSnapshot()

    expect('good bye').toMatchSnapshot()
  })

  it('can do multiple snapshots in a different test', () => {
    expect(1 + 2).toMatchSnapshot()

    expect(1 + 3).toMatchSnapshot()
  })

  it('is not new', () => {
    expect('yes').toMatchSnapshot()
    expect('no').toMatchSnapshot()
  })

  it('has inline snapshot', () => {
    expect('yes').toMatchInlineSnapshot('"yes"')
    expect('no').toMatchInlineSnapshot('"no"')
  })

  it('has snapshot and is known', () => {
    expect('yes').toMatchInlineSnapshot('"yes"')
    expect('no').toMatchInlineSnapshot('"no"')
  })
})
