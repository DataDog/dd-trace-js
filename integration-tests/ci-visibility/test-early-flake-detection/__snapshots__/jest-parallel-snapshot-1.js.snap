// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`parallel snapshot can do multiple snapshots 1`] = `"hello"`;
exports[`parallel snapshot can do multiple snapshots 2`] = `"good bye"`;
exports[`parallel snapshot can do multiple snapshots in a different test 1`] = `3`
exports[`parallel snapshot can do multiple snapshots in a different test 2`] = `4`
exports[`parallel snapshot is not new 1`] = `"yes"`
exports[`parallel snapshot is not new 2`] = `"no"`
exports[`parallel snapshot is flaky 1`] = `3`
exports[`parallel snapshot is flaky 2`] = `"a"`
